packages/thriver-masthead/lib/templates/masthead.html

```diff
@@ -2,7 +2,7 @@
     {{#if isAdmin}}
         {{> sectionAdmin}}
     {{/if}}
-    <section class="masthead" id="{{anchor name=name}}" data-id="{{_id}}">
+    <section class="masthead" id="masthead" data-id="{{_id}}">
```

Comment:
What was the reason for this change?  `{{anchor name=name}}` is used by the sections package to dynamically generate sections on the page, including whether to link them in navigation.  We don't have an ID for the masthead, preventing it from being linked in navigation, but it should remain consistent with the rest of the packages.

If the goal is just to make the element easy to select in masthead.js, just use a different selector:

`const masthead = document.querySelector('section.masthead');`